src/azure-cli/azure/cli/command_modules/servicebus/tests/latest/test_servicebus_alias_commands.py

```diff
@@ -0,0 +1,166 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# AZURE CLI SERVICEBUS - CURD TEST DEFINITIONS
```

Comment:
Minor: Should this be 'CRUD TEST DEFINITIONS'?

Reply:
fixed